README.md

```diff
@@ -123,46 +123,46 @@ more documentation.
 
 ADS-B data from dump1090-docker can be
 [fed to ADS-B Exchange](https://www.adsbexchange.com/how-to-feed) with the help
-of [docker-adsbexchange](https://github.com/webmonkey/docker-adsbexchange).
+of [adsbexchange-docker](https://hub.docker.com/search?q=marcelstoer%2Fadsbexchange&type=image) images.
 
 * Using vanilla Docker
 
   1. Ensure dump1090 is running.
 
-  1. Fetch
-     [docker-adsbexchange](https://github.com/webmonkey/docker-adsbexchange)
-     and build Docker image.
+  1. Run [adsbexchange-docker-feed](https://github.com/marcelstoer/adsbexchange-docker).
 
      ```shell
-     git clone https://github.com/webmonkey/docker-adsbexchange.git
-     cd docker-adsbexchange
-     docker build -t webmonkey/adsbexchange:latest .
+     docker run --rm -d -e "INPUT=decoder:30005" —link dump1090:decoder --name adsbexchange-feed marcelstoer/adsbexchange-docker-feed:latest
      ```
+  1. Run [adsbexchange-docker-mlat](https://github.com/marcelstoer/adsbexchange-docker).
 
-  1. Run [docker-adsbexchange](https://github.com/webmonkey/docker-adsbexchange).
+     **Note**: make sure you replace the dummy values in the command below with your effective values
 
      ```shell
-     docker run --rm -d --link dump1090:decoder --name adsb-exchange [--env RECEIVER_PORT=<port>] webmonkey/adsbexchange:latest
+     docker run --rm -d -e "INPUT=decoder:30005" -e "MLAT_RESULTS=decoder:30104" -e "RECEIVER_LATITUDE=nn.mmmmm" -e "RECEIVER_LONGITUDE=nn.mmmmm" -e "RECEIVER_ALTITUDE=nnnn" -e "RECEIVER_NAME=my-fantastic-ADS-B-receiver" --link dump1090:decoder --name adsbexchange-mlat marcelstoer/adsbexchange-docker-mlat:latest
```

Comment:
Can you wrap this at 80 columns with '\'? If you [view this in GitHub markdown view](https://github.com/jeanralphaviles/dump1090-docker/blob/bc1dc23539708d7ec8ba432bfbc4d07acf4099b5/README.md) you have to scroll to see the whole command.